docs/buildah-build.1.md

```diff
@@ -489,7 +489,7 @@ Sets the configuration for network namespaces when handling `RUN` instructions.
 
 Valid _mode_ values are:
 
-- **none**: no networking;
+- **none**: no networking; Invalid if using --dns, --dns-opt, or --dns-search.
```

Comment:
I think the period and semicolon need to be swapped, for consistency with the list format.